packages/output-handler/src/index.ts

```diff
@@ -63,6 +64,14 @@ const handleTranscriptionSuccess = async (
 	metrics: MetricsService,
 	sourceMediaDownloadUrl: string,
 ) => {
+	const languageCode = LanguageCode.safeParse(transcriptionOutput.languageCode);
```

Comment:
In [`processMessage`](https://github.com/guardian/transcription-service/blob/41b54d899416fe667deb164783a697aab0e7420a/packages/output-handler/src/index.ts#L166) we're parsing the sqs event object before calling this function. It might be cleaner to change [`TranscriptionOutputSuccess`](https://github.com/guardian/transcription-service/blob/41b54d899416fe667deb164783a697aab0e7420a/packages/common/src/types.ts#L75) so that its languageCode is an enum than to do a second round of parsing here.
Although I'm now realising that this would be more of a refactor of our types and zod parsers rather than the original goal of just storing language! So feel free to ignore this or just put in a TODO

Reply:
This opened up a rabbit hole, I've ended up refactoring the language types quite a bit in https://github.com/guardian/transcription-service/pull/118/commits/24780a33ae7ec9ec9b6f867c5a93ae5e483a13e4 

The idea is - going in the type is either a code or auto, coming out either 'unknown' or a valid language code

Reply:
thanks for the refactor! I think your input and output types are way cleaner than what we were doing before